go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ethersphere/bee
 
-go 1.17
+go 1.18
 
 require (
```

Comment:
`go mod tidy` would do wonders to this file

Reply:
fixed